ckanext/data_qld/templates/package/resource_read.html

```diff
@@ -87,60 +87,33 @@
         {% endblock %}
         {% block data_preview %}
           {% if res.datastore_active %}
-              {{ super() }}
+            {{ super() }}
           {% endif %}
         {% endblock %}
       {% endblock %}
     </section>
   {% endblock %}
 
   {% block resource_additional_information_inner %}
```

Comment:
why removal?

Reply:
ckanext-datastore adds this, it should be redundant.

Reply:
Also, the page locations are changing; `package/snippets/dictionary_table.html` will no longer exist. We could update our markup, but from what I can see it's just not necessary.